micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jpa/HibernateMetrics.java

```diff
@@ -177,16 +177,29 @@ public void bindTo(MeterRegistry registry) {
         counter(registry, "hibernate.statements", "The number of prepared statements that were released",
             Statistics::getCloseStatementCount, "status", "closed");
 
-        // Second Level Caching
-        Arrays.stream(statistics.getSecondLevelCacheRegionNames())
-                .forEach(regionName ->  {
+        // Second Level Caching:
+        long numRegions = Arrays.stream(statistics.getSecondLevelCacheRegionNames())
+                .filter(regionName -> hasDomainDataRegionStatistics(regionName))
```

Comment:
Can't this be replaced by a method reference?

```suggestion
                .filter(this::hasDomainDataRegionStatistics)
```